application/models/Permissiontemplates.php

```diff
@@ -291,9 +291,14 @@ public function removeCdataFormat($node)
         return trim(preg_replace($regex_patterns, $regex_replace, $nodeText));
     }
 
+    /**
+     * return if this role GIVE a permission
+     * @param string $sPermission 
+     * @param string $sCRUD 
+     */
     public function getHasPermission($sPermission, $sCRUD)
```

Comment:
did this function must return if current user have permission ?

Reply:
It sounds like it should return boolean value...?

Reply:
yes … :) i update phpDoc

Reply:
:+1: